src/components/movie/MovieDetails.jsx

```diff
@@ -1,10 +1,15 @@
 import { useNavigate, useParams } from "react-router-dom";
-import { GetMoviesBasedOnId, DeleteMovie } from "../../API/movies";
+import { GetMoviesBasedOnId, DeleteMovie, EditMovie } from "../../API/movies";
+import { Link } from "react-router-dom";
+import { MovieSharp } from "@mui/icons-material";
 
 const MovieDetails = () => {
   const { id } = useParams();
   const navigate = useNavigate();
-  const { movies: movie, isPending, error } = GetMoviesBasedOnId(id);
+  const { movies: movie, error } = GetMoviesBasedOnId(id);
+
+  const handleEdit = () => {
+  };
```

Comment:
where is edit functionality?

Reply:
Please also a one line space between functions

Reply:
Edit functionality is implemented in the Edit component, I forgot to remove this function from here, now this is updated.